twake/backend/node/src/services/preview/src/previews/services/mime.ts

```diff
@@ -0,0 +1,1228 @@
+export const officeExtensions = [
```

Comment:
It will be better to have the functions in a file and all your contants in another one